packages/web_drivers/lib/chrome_driver_installer.dart

```diff
@@ -43,29 +50,43 @@ class ChromeDriverInstaller {
         await runDriver();
       }
     } finally {
-      driverDownload?.deleteSync();
+      // Only delete if the user is planning to override the installs.
+      // Keeping the existing version might make local development easier.
```

Comment:
Also if a CI build runs multiple `felt` commands using an existing version speeds up the build.

Reply:
thanks added.